### DISCUSSION TITLE
Coolify Server Response Time is Slow

### DISCUSSION THREAD

**Discussion by parisosuch** [coollabsio/coolify#3096] [Q&A]
I am self hosting a coolify instance. The overall ping response time is low, but when cold loading the webserver, it takes almost 7s to load the fonts? Everything else loads fairly quickly. Maybe its a hardware issue but i'm curious if you guys know anything?

Server specs:
- Ubuntu LTS 22
- OS is running on 128GB SSD
- 16gb RAM
- Quad-core 3.7ghz i5

Maybe there is a bottleneck in my network but why would only the fonts take so long to load?

![image](https://github.com/user-attachments/assets/6748e6f1-c000-4420-821a-42447775a3a9)

**Comment by LEstradioto** [comment]
weird, Ive got normal like 30-50ms on assets (ec2 ubuntu 8gb) did you try to clean cache? stop extensions, antivirus, etc...

**Comment by peaklabs-dev** [comment]
@parisosuch-dev I am not sure if this is coolify related, please try clearing cache (like LEstradioto said) and incognito mod and test again?

**Comment by medsghiri** [comment]
I'm currently dealing with the same problem, I switched from railway to my own VPS. I'm hosting a Next.js website and build using nixpacks
Reduce initial server response time Root document took 920 ms
Reduce initial server response time Root document took 1000 ms

4 vCPU Cores
6 GB RAM
150 GB NVMe

**Comment by alihammad99** [comment]
I have deployed a Next.js app without panel, and it's very fast.

In the recent project, I decided to use Coolify, it worked, but it's very very slow when loading the page, it takes 10s just to redirect when click a link

**Reply by stickerdaniel** [reply]
I have the same problem - the front end is really slow. Have you been able to fix this somehow?

**Reply by alihammad99** [reply]
Yes, I just removed Coolify and used Caddy for proxy management

**Comment by redaboucetta0** [comment]
That interesting mee too i m hosting with coolify jellyfin to stream media personel, its slow so slow

**Comment by stickerdaniel** [comment]
When I access the dashboard with the server IP directly, it is as fast as expected.

**Comment by KishinNext** [comment]
Same here, I'm running Coolify in a VPS:

CPU: core 2 (CPU usage 14%)
Memory: 8 GB (Memory Usage 18%)

Any solution for this? :C

**Reply by parisosuch** [reply]
It was a networking issue for me. Was running a double NAT.

**Reply by redaboucetta0** [reply]
for me i restar coolify and that work

**Reply by sukhrobabdullaev** [reply]
the same issue with me, I restart coolify, it is working fine, but sometimes server response is 504

**Comment by reyemtm** [comment]
Running a very basic fastify app on coolify on a (budget) 10 core vps and a basic index.html landing page with a few images is so slow like 10s response times, not sure what is going on. The load time should be pretty much instant like < 500ms or less, though I did not test the server with for ex a standalone nginx and html, which is maybe what I should do to get a baseline. Still seems like something is off somewhere.

**Comment by sahaskamble** [comment]
Same thing is happening with me coolify is just tooo slow and reading all this comments I guess this is some internal thing like 10s to 20s just to go back like WTF is happening and I have used coolify in past and it worked like a charm but now it's becoming hell for me 
I just decided to move to dokploy it looks promising lets see

**Comment by Coderamrin** [comment]
i just installed coolify on my vps, it's so slow. 
i can't even navigate through the pages!

**Reply by sahaskamble** [reply]
I just switched to dokploy now but I guess that slowness is because of vps is not in closest zone check it
